gnucash/report/html-piechart.scm

```diff
@@ -53,13 +53,17 @@
   (record-constructor <html-piechart>))
 
 (define (gnc:make-html-piechart)
+  (issue-deprecation-warning
+   "(gnc:make-html-piechart) is deprecated. use gnc:make-html-chart instead.")
   (gnc:make-html-piechart-internal '(pixels . -1) '(pixels . -1) #f #f #f #f #f #f #f #f #f #f #f))
 
 (define gnc:html-piechart-data
   (record-accessor <html-piechart> 'data))
 
-(define gnc:html-piechart-set-data!
-  (record-modifier <html-piechart> 'data))
+(define (gnc:html-piechart-set-data! pie newdata)
+  (if (gnc:html-chart? pie)
+      (issue-deprecation-warning "uhhh wrong api.")
```

Comment:
Uhhh, horrible deprecation warning :(
For all the reasons I have mentioned earlier: this only tells the user they did something wrong, but it won't give them a clue as to how to fix it.

Reply:
forgot this one